code/modules/admin/topic.dm

```diff
@@ -773,9 +689,9 @@
 		if(!check_rights(R_SPAWN))
 			return
 
-		var/mob/living/carbon/human/H = locate(href_list["makeai"])
-		if(!istype(H))
-			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human.", confidential = TRUE)
+		var/mob/our_mob = locate(href_list["makeai"])
```

Comment:
make sure to verify that this is actually a mob. Admins might press the button on a delete mob and it'll cause a runtime